appinfo/info.xml

```diff
@@ -3,7 +3,15 @@
 	  xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/info.xsd">
 	<id>contacts</id>
 	<name>Contacts</name>
-	<description>The new and improved app for your Contacts.</description>
+	<summary>Contacts GUI for Nextcloud’s CardDAV server</summary>
```

Comment:
This is much too technical. :) The first sentence of the description is much better.